src/tab.rs

```diff
@@ -17,6 +17,15 @@ pub fn render_tab(text: String, tab: &TabInfo, user_conf: UserConfiguration) ->
     } else {
         text
     };
+    // Tab index is not necessarily tab position
+    let text = text
+        .split_once("Tab #")
+        .and_then(|(_, raw_index)| {
+            let tab_name_index = raw_index.parse::<u32>().ok()?;
+            (text == format!("Tab #{tab_name_index}")).then_some(user_conf.default_tab_name)
```

Comment:
The code doesn't build. Change the line 16 to `user_conf.default_tab_name.clone()`.